au4/src/trackedit/internal/trackeditactionscontroller.cpp

```diff
@@ -181,6 +200,32 @@ void TrackeditActionsController::paste()
     }
 }
 
+void TrackeditActionsController::clipSplit(const ActionData& args)
+{
+    ClipKey clipKey = args.arg<ClipKey>(0);
+    if (!clipKey.isValid()) {
+        return;
+    }
+
+    secs_t playbackPosition = globalContext()->playbackState()->playbackPosition();
+
+    trackeditInteraction()->split(clipKey.trackId, playbackPosition);
```

Comment:
It looks like we should just resend the TRACK_SPLIT action with the track id argument here